recipes/simpleconf/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "python-simpleconf" %}
+{% set version = "0.5.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python-simpleconf-{{ version }}.tar.gz
+  sha256: e95f811f66beadc62d2d1b1bce680bebf6261af692bc94a608c3f29051da5e5a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry >=0.12
+  run:
+    - diot >=0.1,<0.2
+    - python >=3.6,<4.0
+    - poetry >=0.12
```

Comment:
Is poetry required at runtime, or it just install?

(I can't seem to view the CI runs today to see what the issue was)

Reply:
poetry required at the building stage. 
let me remove this from run section